webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -90,6 +90,17 @@ def timestamp_to_date(
 
 
 @library.filter
+def time_ago(dt, format='%a, %b %d %H:%M %Z'):
+    return jinja2.Markup(
+        '<time datetime="{}" class="ago">{}</time>'
+        .format(
+            dt.isoformat(),
+            dt.strftime(format)
+        )
+    )
+
+
```

Comment:
Well I just checked and we of course use in the Tokens page, to indicate when a token will expire! :)